integration-tests/static-buffer/README.md

```diff
@@ -0,0 +1,18 @@
+# Static buffer configuration demo
+
+This is a dummy contract illustrating how the [static buffer](/ARCHITECTURE.md#communication-with-the-pallet)
+can be be configured using the environmental variables.
+
+Simply, run:
+```bash
+cargo clean
+STATIC_BUFFER_SIZE=30 cargo test -F e2e-tests
+```
+
+This will configure the buffer to have enough space to instantiate the contract,
+but not enough space to retrieve the caller's address as it is of 32 bytes,
+but we only allocated 30 bytes to the contract.
+
+## Note
+You must run `cargo clean` every time you want to modify the buffer size
```

Comment:
It is a sad property of so nice solution=)

We need to highlight that in the documentation

Reply:
Also, we need to notify all projects that provide deployment of the contracts to the network(like Swanky, Patron, etc). Because they should somehow support `STATIC_BUFFER_SIZE` during deployment, and they need to be sure that the binary uses the correct size for the buffer.

Reply:
That's why I am personally in favour of #1872 since it is easier and more idiomatic for ppl to use, i.e. it eliminated any issues associated with the host machine.